bears/c_languages/ArtisticStyleBear.py

```diff
@@ -0,0 +1,866 @@
+
+                int Foo(bool isBar)
+                {
+                        if (isFoo) {
+                                bar();
+                                return 1;
+                        } else
+                                return 0;
+                }
+
+            For example: Horstmann style uses broken braces and run-in
+            statements. Switches are indented to allow a run-in to the opening
+            switch block. This style frequently is used with an indent of 3
+            spaces. If set to ``horstmann``, prefer::
+
+                int Foo(bool isBar)
+                {   if (isBar)
+                    {   bar();
+                        return 1;
+                    }
+                    else
+                        return 0;
+                }
+
+            For example: Google style uses attached braces and indented class
+            access modifiers. This is not actually a unique brace
+            style, but is Java style with a non-brace variation. This style
+            frequently is used with an indent of 2 spaces. If set to
+            ``google``, prefer::
+
+                int Foo(bool isBar) {
+                    if (isBar) {
+                        bar();
+                        return 1;
+                    } else
+                        return 0;
+                }
+
+            For example: Mozilla style uses linux braces. Opening braces are
+            broken from classes, structs, enums, and function definitions. The
+            braces are attached to everything else, including namespaces,
+            arrays, and statements within a function. This style frequently is
+            used with an indent of 2 spaces. If set to ``mozilla``, prefer::
+
+                int Foo(bool isBar)
+                {
+                    if (isBar) {
+                        bar();
+                        return 1;
+                    } else
+                        return 0;
+                }
+
+            For example: Pico style uses broken braces and run-in statements
+            with attached closing braces. The closing brace is attached to the
+            last line in the block. Switches are indented to allow a run-in to
+            the opening switch block. This style frequently is used with an
+            indent of 2 spaces. If set to ``pico``, prefer::
+
+                int Foo(bool isBar)
+                {   if (isBar)
+                    {   bar();
+                        return 1; }
+                    else
+                        return 0; }
+
+            For example: Lisp style uses attached opening and closing braces.
+            The closing brace is attached to the last line in the block.
+            If set to ``lisp``,
+            prefer::
+
+                int Foo(bool isBar) {
+                if (isBar) {
+                    bar()
+                    return 1; }
+                else
+                    return 0; }
```

Comment:
I actually don't have something against this longer documentation, but we definitely need some header line that summarizes the possible values in a few sentences :+1: